src/store/reducers/walletConnectionsCache/walletConnectionsCache.test.ts

```diff
@@ -30,9 +31,9 @@ describe("walletConnectionsCacheSlice", () => {
   it("should handle setWalletConnectionsCache", () => {
     const connections: ConnectionData[] = [
       {
-        id: 2,
+        id: "2",
         name: "Wallet name #2",
-        owner: "Yoroi",
+        selectedAid: "Yoroi",
```

Comment:
Can we update these to proper AIDs instead of e.g. "Yoroi" in case it causes confusion later